managed/services/server/server.go

```diff
@@ -908,7 +897,15 @@ func isAgentsStateUpdateNeeded(mr *serverpb.MetricsResolutions) bool {
 	return true
 }
 
-// check interfaces
+// check interfaces.
 var (
 	_ serverpb.ServerServer = (*Server)(nil)
 )
+
+func canUpdateDurationSetting(setting, envSetting time.Duration) bool {
```

Comment:
Great function name!

Can we rename `setting` and `envSetting` to highlight the difference between them. As of now, it's not apparent.